controllers/apply/lib/field-types/address.js

```diff
@@ -10,16 +10,16 @@ class AddressField extends Field {
 
         this.type = 'address';
 
-        if (!props.explanation) {
-            this.explanation = this.localise({
-                en: `<p>Enter the postcode and search for the address, or enter it manually below.`,
-                cy: `Rhowch y cod post a chwiliwch am y cyfeiriad, neu ei deipio isod.`
-            });
-        }
+        this.isRequired = true;
+
+        this.explanation = this.localise({
```

Comment:
This explanation is a core part of the address field, so have made it static. We don't override it anywhere.